manifests/params.pp

```diff
@@ -23,8 +23,10 @@
   if str2bool($::is_pe) {
```

Comment:
https://forge.puppet.com/puppetlabs/stdlib#is_pe says that this function is no longer useful after PE3.x. https://tickets.puppetlabs.com/browse/MODULES-2279 confirms that. This entire branch is useless as that support was dropped long ago and can be removed. I don't know how to make a suggestion that via github to remove it that fixes the indents of the section following.